app/client/components/user/CreateUser.tsx

```diff
@@ -0,0 +1,143 @@
+import React  from 'react';
+import { connect } from 'react-redux';
+
+import { createUser, RESET_CREATE_USER } from '../../reducers/userReducer'
+import { StyledButton, ModuleHeader } from '../styledElements'
+import styles from './styles.module.css';
+import CreateUserForm, {ICreateUser} from './CreateUserForm';
+import CreateVendorForm, {ICreateVendor} from './CreateVendorForm';
+
+//TODO(ts): fix types
+interface DispatchProps {
+  createUser: (body: any) => void,
+  resetCreateUser: () => void
+}
+
+interface StateProps {
+  users: any
+}
+
+interface OuterProps {
+  isVendor: boolean
+}
+
+declare global {
+  interface Window {
+    BENEFICIARY_TERM: string
+    IS_USING_BITCOIN: boolean
+    MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT: number
+    CURRENCY_NAME: string
+  }
+}
+
+type Form = ICreateUser & ICreateVendor
+type Props = DispatchProps & StateProps & OuterProps
+
+class CreateUser extends React.Component<Props> {
```

Comment:
fyi this is how you can tell what props there are, and whether they're from redux state/dispatch or passed into the component (outer)